### ISSUE TITLE
prometheus api and +/-Inf values

### ISSUE THREAD

**Issue by datamuc** [influxdata/influxdb#10490]
__System info:__ [Include InfluxDB version, operating system name, and other relevant details

InfluxDB v1.7.0 (git: 1.7 dac4c6f571662c63dc0d73346787b8c7f113222a)

__Steps to reproduce:__

Send +/-Inf values with prometheus' remote_write.

__Expected behavior:__ [What you expected to happen]

Ignore +/-Inf values

__Actual behavior:__ [What actually happened]

The whole batch fails.

__Additional info:__ [Include gist of relevant config, logs, etc.]

Here is an example setup with docker-compose:

https://gist.github.com/datamuc/d6085cc4c4190619796f0ea5a8661dce

```
influxdb_1       | [httpd] 192.168.208.3 - admin [16/Nov/2018:16:34:55 +0000] "POST /api/v1/prom/write?db=prom HTTP/1.1" 400 59 "-" "Go-http-client/1.1" 8ce96825-e9bd-11e8-8001-0242c0a8d002 88584
prometheus_1     | level=warn ts=2018-11-16T16:34:55.788646973Z caller=queue_manager.go:531 component=remote queue="0:http://influxdb:8086/api/v1/prom/write?db=prom" msg="Error sending samples to remote storage" count=375 err="server returned HTTP status 400 Bad Request: {\"error\":\"+/-Inf is an unsupported value for field value\"}"
```

All metrics sent in this POST request are lost because of a single +Inf value.

**Comment by datamuc** [comment]
QuickFix, at least identifying the culprit:

```diff
diff --git a/prometheus/converters.go b/prometheus/converters.go
index f3f438587..3cb9a5064 100644
--- a/prometheus/converters.go
+++ b/prometheus/converters.go
@@ -57,6 +57,7 @@ func WriteRequestToPoints(req *remote.WriteRequest) ([]models.Point, error) {
 		for _, s := range ts.Samples {
 			// skip NaN values, which are valid in Prometheus
 			if math.IsNaN(s.Value) {
+				// FIXME: Add some logging here?
 				droppedNaN = ErrNaNDropped
 				continue
 			}
@@ -66,7 +67,8 @@ func WriteRequestToPoints(req *remote.WriteRequest) ([]models.Point, error) {
 			fields := map[string]interface{}{fieldName: s.Value}
 			p, err := models.NewPoint(measurement, models.NewTags(tags), fields, t)
 			if err != nil {
-				return nil, err
+				// FIXME: Add some logging here?
+				continue
 			}
 
 			points = append(points, p)
```

**Comment by marlon-nc** [comment]
This is tripping me up in 1.7.4 as well

**Comment by e-dard** [comment]
Fixed via #12813
